DataFormats/Scouting/README.md

```diff
@@ -0,0 +1,12 @@
+#  DataFormats/Scouting
+
+## Scouting Data Formats
+
+Any changes to the Scouting data formats must be backwards compatible. In order to ensure the Scouting formats can be read by all future CMSSW releases, there is a `TestRun3ScoutingDataFormats` unit test, which makes use of the `TestReadRun3Scouting` analyzer and the `TestWriteRun3Scouting` producer. The unit test checks that the objects can be read properly from
+
+* a file written by the same release
+* files written by (some) earlier releases
+
+If the persistent format of any Scouting data format gets changed in the future, please adjust the `TestReadRun3Scouting` and `TestWriteRun3Scouting` modules accordingly. It is important that every member container has some content in this test. Please also add a new file to the [https://github.com/cms-data/DataFormats-Scouting/](https://github.com/cms-data/DataFormats-Scouting/) repository, and update the `TestRun3ScoutingDataFormats` unit test to read the newly created file. The file name should contain the version numbers of the data format classes (from classes_def.xml) in alphabetical order and the release or pre-release with which it was written. If the latest file of Run 3 scouting before the update has not been used in data taking, the file can be deleted.
```

Comment:
It wouldn't be bad to also rearrange classes_def.xml in order to have the classes listed there in alphabetical order, in order to avoid some head hache to who's trying to associate the versions in the name of the data file to the classes.

Reply:
Thanks @perrotta for the suggestion, I agree a matching order in the `classes_def.xml` would help. @wddgit Could you take care of it (either as part of the Run2 scouting PR, or in a separate PR)?

Reply:
I'll include it in the Run 2 Scouting PR which I am currently implementing. A good idea.